--- origsrc/libuninameslist/Makefile.in	2009-12-31 14:50:23.000000000 -0600
+++ src/libuninameslist/Makefile.in	2010-03-02 19:42:15.521967200 -0600
@@ -42,7 +42,7 @@ install: both
 	$(LIBTOOL) --mode=install $(INSTALL) -c libuninameslist-fr.la $(DESTDIR)$(libdir)/libuninameslist-fr.la
 	$(LIBTOOL) --finish $(DESTDIR)$(libdir)
 	mkdir -p $(DESTDIR)$(incdir)
-	$(INSTALL_DATA) nameslist.h $(DESTDIR)$(incdir)/uninameslist.h
+	$(INSTALL_DATA) $(srcdir)/nameslist.h $(DESTDIR)$(incdir)/uninameslist.h
 
 clean:
 	-rm -f *.lo *.o *.la
